### PR TITLE
Jason test

### PR DESCRIPTION
**Pull request by jsonpittman**
Item page is happier

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (csdear) [inline test.js:1]: @jsonpittman  Got some artifacts here.

File: templates/views/items.pug [hunk 2]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (csdear) [inline templates/views/items.pug]: @jsonpittman

**Comment by fk1018** [comment]
In this pr it looks like most of the changes are to the vendor library for bootstrap. I am not sure we want to keep a copy of boostrap inside of the github repo. There are some build tools and package managers I think we can leverage to help us remove the bootstrap source files from our code repository if this is the direction we may be wanting to go. Any thoughts @csdear @jsonpittman ?

**Comment by csdear** [comment]
@fk1018 @jsonpittman  You mean like a CDN for bootstrap?  That's fine by me!

**Comment by csdear** [comment]
@fk1018 I remember us talking a little about webpack the other day-- is that more what you are thinking?  If so I would the vendor library would be in there, just bundled up to something like a bundle.js file at build?   Maybe you can explain it to me, i like webpack enough but have difficulty understanding it... it can get deceptively complicated... which is not to say I don't want to use it, i do, I just wish I understood it better!

**Comment by fk1018** [comment]
We could definately use web-pack to build out the files if we want them moved into a different location. I was thinking instead of manually including boostrap in the repo we isntall it on the server at runtime. So using bootstrap from npm or bower or some package manager. And when we deploy / on the deployment server it will download a copy of it then instead of us manually pushing the bootstrap source code with our code repository. @csdear

**Pull request merged**